### PR TITLE
Dethanga edits

### PR DESCRIPTION
**Pull request by dethanga**
Hi Anne,

As per your request, I have replaced the "labID" to "redirectTo" and "devnet-alexa-lab" to "ucsm-powertool-101", respectively.

Kindly check.

### CODE REVIEW

File: labs/devnet-alexa-lab/devnet-alexa-lab.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "labId": "devnet-alexa-lab",
+  "redirectTo": "ucsm-powertool-101",
```
Comment (annegentle) [inline labs/devnet-alexa-lab/devnet-alexa-lab.json:3]: Keep the labId line, and add the redirectTo as you did.

**Comment by annegentle** [comment]
Fixes https://github.com/CiscoDevNet/learning-labs-issues/issues/142

File: labs/devnet-alexa-lab/devnet-alexa-lab.json (hunk 1 above)
Comment (dethanga) [reply labs/devnet-alexa-lab/devnet-alexa-lab.json:3]: Thanks Anne, merging the changes and closing this pull request.

**Comment by annegentle** [comment]
Hi @dethanga and cc @klevenstein  John does not want to redirect this Lab after all - he is working on edits after seeing that people were still doing this lab even with their own equipment.

**Comment by dethanga** [comment]
@annegentle and cc @klevenstein 
Hi Anne, Thanks for your notes. Should I remove the added "redirect ID" from the JSON file? 
So that I will revert back changes to older copy and merge to the master. Please advise.

**Pull request merged**